1.8/administration/id-and-access-mgt/iam-api.md

```diff
@@ -4,28 +4,28 @@ post_title: Authentication HTTP API Endpoint
 
 You can make external calls to HTTP API endpoints in your DC/OS cluster.
 
-You must first obtain an HTTP API token and then include it in your HTTP request.
+You must first obtain an authentication token and then include it in your HTTP request.
 
-HTTP API tokens expire after 5 days. You can view the expiration time in the ["exp" (Expiration Time) Claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) of the JSON Web Token (JWT). You can refresh your token by re-logging in to DC/OS.
+Authentication tokens expire after 5 days. You can view the expiration time in the ["exp" (Expiration Time) Claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) of the JSON Web Token (JWT). You can refresh your token by re-logging in to DC/OS.
```

Comment:
> Authentication tokens expire after 5 days. 

Since this is configurable, maybe we should say "By default, authentication tokens expire after 5 days."?